src/cargo/core/compiler/job_queue.rs

```diff
@@ -783,8 +784,7 @@ impl<'cfg> DrainState<'cfg> {
                 match result {
                     Ok(()) => self.finish(id, &unit, artifact, cx)?,
                     Err(error) => {
-                        let msg = "The following warnings were emitted during compilation:";
-                        self.emit_warnings(Some(msg), &unit, cx)?;
```

Comment:
Note: this is a change of behavior

Reply:
Is this necessary? AFAIK there's no such message for regular compilation errors.

Reply:
My intention wasn't to say "this is a bad change of behavior" but to just raise awareness of how the message is changing.  We used to have an introductory statement in this case but will no longer.  I felt this would be worth highlighting when this gets a wider view during FCP